packages/editor-ui/src/types/folders.types.ts

```diff
@@ -0,0 +1,18 @@
+import type { ITag } from '@/Interface';
+import type { ProjectSharingData } from './projects.types';
+
+export type FolderShortInfo = {
+	id: string;
+	name: string;
+};
+
+export type Folder = {
```

Comment:
can we dedupe the types here and `Interface` and `ResourceListLayout`?

Reply:
Definitely

Reply:
this is still here? Could we reusue this in interface so it's deduped?